test/e2e/conftest.py

```diff
@@ -215,10 +268,10 @@ def session_worker(
 @pytest.fixture(scope="class")
 def class_worker(
     request: pytest.FixtureRequest,
-    worker_config: DeadlineWorkerConfiguration,
+    worker_config_session: DeadlineWorkerConfiguration,
     ec2_worker_type: Type[EC2InstanceWorker],
 ) -> Generator[DeadlineWorker, None, None]:
-    with create_worker(worker_config, ec2_worker_type, request) as worker:
+    with create_worker(worker_config_session, ec2_worker_type, request) as worker:
         yield worker
```

Comment:
We have a class-scoped worker fixture? Is this actually being used anywhere? If so, we'll need a fleet for this as well